runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/status/BeamWorkerStatusGrpcService.java

```diff
@@ -187,4 +194,17 @@ private String getStatusErrorOrInfo(WorkerStatusResponse response) {
         ? response.getError()
         : response.getStatusInfo();
   }
+
+  private String handleAndReturnExceptionResponse(Exception e) {
+    LOG.warn(DEFAULT_EXCEPTION_RESPONSE, e);
+    if (e instanceof InterruptedException) {
+      Thread.currentThread().interrupt();
+    }
+    StringBuilder response = new StringBuilder();
+    response.append(DEFAULT_EXCEPTION_RESPONSE).append(":").append(e.getClass().getCanonicalName());
```

Comment:
Add space after `:` here and the line below otherwise the message will look awkwardly formatted.

Reply:
done.